db/migrate/20221005133322_add_collected_by_to_sample_metadata.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Adds the column: collected_by, to the sample_metadata table
+# To allow for manifests uploading samples from multiple collection sites
+class AddCollectedByToSampleMetadata < ActiveRecord::Migration[6.0]
+  def change
+    add_column :sample_metadata, :collected_by, :string
```

Comment:
I know most of our columns don't have descriptions, but it's something I think we could improve on. I wonder if we could add a description? The description of the equivalent field in EBI is 'name of persons or institute who collected the specimen', which seems alright?